prow/config.yaml

```diff
@@ -14752,6 +14752,35 @@ periodics:
     - name: docker-graph
       emptyDir: {}
 
+- interval: 24h
```

Comment:
Can we run a few times a day? Maybe every 6 hours? (That's what we had periodic tap tests running at. Then you at least get another run in on the same day.)

Reply:
Done